### PR TITLE
Some small fixes to the fcp library

### PR DESCRIPTION
**Pull request by debbiedub**
These changes are according the comments on pull request #2.

All comments are addressed except the creation of a test case that is not included.

### CODE REVIEW

File: src/net/pterodactylus/fcp/FcpConnectionHandler.java [hunk 1]
```diff
@@ -31,6 +32,9 @@
  */
 class FcpConnectionHandler implements Runnable {
 
+	/** Logger. */
+	private static final Logger logger = Logger.getLogger(FcpConnection.class.getName());
```
Comment (Bombe) [inline src/net/pterodactylus/fcp/FcpConnectionHandler.java]: The class the logger is associated with should always be the class in which the logger is declared and used (unless you have very specific reasons).

File: src/net/pterodactylus/fcp/FcpMessage.java [hunk 2]
```diff
@@ -37,6 +38,9 @@
  */
 public class FcpMessage implements Iterable<String> {
 
+	/** Logger. */
+	private static final Logger logger = Logger.getLogger(FcpConnection.class.getName());
```
Comment (Bombe) [inline src/net/pterodactylus/fcp/FcpMessage.java]: Same here.

File: src/net/pterodactylus/fcp/FcpConnectionHandler.java (hunk 1 above)
Comment (debbiedub) [inline src/net/pterodactylus/fcp/FcpConnectionHandler.java]: Oh, no! I was too quick with the cut'n'paste.

Thanks for noticing! Good think you are looking at this.

**Comment by debbiedub** [comment]
I have now put the corrections of the logger classes in the first commit.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/bin
```
Comment (Bombe) [inline .gitignore]: Well, this still doesn’t belong here.

**Comment by debbiedub** [comment]
This pull request is amended. :)

**Comment by Bombe** [comment]
Okay, I’m not sure why I haven’t mentioned that before but: this repository is a stale, almost 8-year old copy of https://github.com/Bombe/jFCPlib, the official jFCPlib repository. Some of the issues of this pull request have already been addressed in the `master` branch, such as the listener management. Others, such as the bug in the `copy` routine have been applied to `next`.

You are welcome to continue participating at the canonical repository. Also, check out the `feature/quelaton-client` branch which holds a new FCP client that aims to be a lot easier to use than `FcpConnection` and friends. :)

**Pull request closed without merging**